host/sgx/create.c

```diff
@@ -1004,8 +1033,16 @@ oe_result_t oe_create_enclave(
     enclave->ocalls = (const oe_ocall_func_t*)ocall_table;
     enclave->num_ocalls = ocall_count;
 
-    /* Invoke enclave initialization. */
+/* Invoke enclave initialization. */
+#ifdef OE_WITH_EXPERIMENTAL_EEID
+    oe_enclave_with_config_t enclave_with_config;
+    enclave_with_config.enclave = enclave;
+    enclave_with_config.config.data = context.config.data;
+    enclave_with_config.config.size = context.config.size;
+    OE_CHECK(_initialize_enclave_with_config(&enclave_with_config));
```

Comment:
Adding a new ECALL makes the code more complex and harder to use, so for now I haven't added one. Since it would change the OE API/ECALL interface, this may also require a new design document and a public discussion.